.github/scripts/trymerge.py

```diff
@@ -413,6 +414,16 @@ class WorkflowCheckState(NamedTuple):
 CIFLOW_TRUNK_LABEL = re.compile(r"^ciflow/trunk")
 MERGE_RULE_PATH = Path(".github") / "merge_rules.yaml"
 
+BOT_AUTHORS = ["github-actions", "pytorchmergebot", "pytorch-bot"]
+
+LABEL_ERR_MSG_TITLE = "This PR needs a label"
+LABEL_ERR_MSG = (
+    f"# {LABEL_ERR_MSG_TITLE}\n"
+    "If your changes are user facing and intended to be a part of release notes, please use a label starting with `release notes:`.\n\n"  # noqa: E501  pylint: disable=line-too-long
+    "If not, please add the `topic: not user facing` label.\n\n"
+    "For more information, see https://github.com/pytorch/pytorch/wiki/PyTorch-AutoLabel-Bot#why-categorize-for-release-notes-and-how-does-it-work."  # noqa: E501  pylint: disable=line-too-long
+)
```

Comment:
Thinking about how the formatting will look when there's a merge failed, with the current approach we'll end up with two H1 heading lines back to back (here's [an example](https://github.com/pytorch/pytorch/pull/92107#issuecomment-1382241578) of how the comment looks).  To fix that, the "This PR needs a label" line shouldn't be treated as a header when it's failing a merge

We could take something like the following and stick `MERGE_LABEL_ERR_MSG` in the merge failure message

```suggestion
LABEL_ERR_MSG_TITLE = "This PR needs a label"
LABEL_ERR_MSG_BODY = (
    "If your changes are user facing and intended to be a part of release notes, please use a label starting with `release notes:`.\n\n"  # noqa: E501  pylint: disable=line-too-long
    "If not, please add the `topic: not user facing` label.\n\n"
    "For more information, see https://github.com/pytorch/pytorch/wiki/PyTorch-AutoLabel-Bot#why-categorize-for-release-notes-and-how-does-it-work."  # noqa: E501  pylint: disable=line-too-long
)
LABEL_ERR_MSG = (
    f"# {LABEL_ERR_MSG_TITLE}\n"
    f"{LABEL_ERR_MSG_BODY}"
)
MERGE_LABEL_ERR_MSG = (
    f"{LABEL_ERR_MSG_TITLE}\n"
    f"{LABEL_ERR_MSG_BODY}"
)
```

Reply:
Ah, great catch and thanks for sharing the comment example! I was wondering if there's a way to actually visualise the bot comments on GH.

Reply:
The way I do it, I find a random merge comment on github and then apply the edits my change *should* make and see how it ends up looking